### PR TITLE
Add Alder Lake configuration for hwaccel

### LINKED ISSUE #3170: [Support]: Need libmfxgen1 library installed to support Alder Lake CPU QSV transcoding
### Describe the problem you are having

I have an Alder Lake gen12 CPU that I'm utilizing QuickSync for hardware transcoding.

In order to use the qsv device, the Docker image needs to have libmfxgen1 installed as per the [Intel compatibility matrix](https://github.com/Intel-Media-SDK/MediaSDK)

The stock Docker image will fail to initialize the hardware encoder/decoders using qsv if this package is not installed (see [general Ubuntu bug](https://bugs.launchpad.net/ubuntu/+source/ffmpeg/+bug/1970637))

After adding the[ Intel repository](deb [arch=amd64] https://repositories.intel.com/graphics/ubuntu focal main) and installing libmfxgen1 via apt-get results in the ffmpeg instance being able to correctly utilize qsv.

### Version

0.10.1-83481AF

### Frigate config file

```yaml
ffmpeg:
        hwaccel_args:
                - -hwaccel
                - qsv
                - -qsv_device
                - /dev/dri/renderD128
        output_args:
          record: -f segment -segment_time 10 -segment_format mp4 -reset_timestamps 1 -strftime 1 -c:v h264_qsv -c:a copy -an
        input_args:
                - -hide_banner
                - -avoid_negative_ts
                - make_zero
                - -fflags
                - +genpts+discardcorrupt+igndts
                - -rtsp_transport
                - tcp
                - -stimeout
                - 5000000
                - -analyzeduration
                - 2147483647
                - -probesize
                - 2147483647
```

### Relevant log output

```shell
Error initializing an internal MFX session: unsupported (-3)
```

### FFprobe output from your camera

```shell
.
```

### Frigate stats

_No response_

### Operating system

Other Linux

### Install method

Docker Compose

### Coral version

M.2

### Network connection

Wired

### Camera make and model

.

### Any other information that may be helpful

_No response_

### PR DESCRIPTION
**Pull request by charlesmunger**
Fixes #3170

### CODE REVIEW

**Comment by NickM-27** [comment]
Is this true for 0.10 or are these tested with 0.11? If the RC then this should point towards `release-0.11.0` branch

File: docs/docs/configuration/hardware_acceleration.md [hunk 1]
```diff
@@ -41,7 +45,7 @@ ffmpeg:
     - yuv420p
 ```
 
-### Intel-based CPUs (>=10th Generation) via Quicksync
+### >=10th Generation via Quicksync
```
Comment (NickM-27) [inline docs/docs/configuration/hardware_acceleration.md:48]: I think this should be ####

File: docs/docs/configuration/hardware_acceleration.md [hunk 2]
```diff
@@ -52,6 +56,31 @@ ffmpeg:
     - /dev/dri/renderD128
 ```
 
+### >=12th Generation via Quicksync
```
Comment (NickM-27) [inline docs/docs/configuration/hardware_acceleration.md:59]: Same here (####)

**Review by NickM-27 — CHANGES_REQUESTED** [review]

**Review by NickM-27 — APPROVED** [review]

File: docs/docs/configuration/hardware_acceleration.md (hunk 2 above)
Comment (charlesmunger) [reply docs/docs/configuration/hardware_acceleration.md]: Done

File: docs/docs/configuration/hardware_acceleration.md (hunk 1 above)
Comment (charlesmunger) [reply docs/docs/configuration/hardware_acceleration.md]: Done

**Comment by charlesmunger** [comment]
> Is this true for 0.10 or are these tested with 0.11? If the RC then this should point towards `release-0.11.0` branch

Done.

**Comment by charlesmunger** [comment]
So, I think this probably shouldn't be merged as is... or at least more discussion is needed. With a variety of tuning parameters, I've been able to get the camera-to-live-view latency of the QSV decoder down to about 2.5 seconds - and at least for small low framerate video, I did not see any CPU usage improvement vs software decoding. 

With vaapi, I was able to get the same excellent latency as the software decoder, but it _increased_ CPU usage by about half. I suspect that unless hardware accelerated scaling filters are used, hardware acceleration for substreams isn't worthwhile on fast CPUs, because of the cost of copying raw frame data out of GPU memory. And even for people who are downscaling their detect streams, I bet that `lowres 1` or `lowres 2` will probably be quite efficient.

It's easy to imagine a setup where video is decoded in hardware, then motion detection is run via opencv's OpenCL kernels directly, with only frames destined for snapshots or inference going to main memory, but that would be a whole lot more complexity and coupling to hardware, plus you'd likely lose process isolation between ffmpeg and the python process.

**Pull request closed without merging**